torch/library.py

```diff
@@ -553,6 +554,82 @@ def register(func):
         return register(func)
 
 
+def register_autograd(op: _op_identifier, setup_context_fn: Callable, backward_fn: Callable, /, *, lib=None) -> None:
+    r"""Register a backward formula for this custom op.
+
+    In order for an operator to work with autograd, you need to register
+    a backward formula. There are two pieces to this:
+    1. You must tell us what we need to save from the forward pass for
+    the backward pass. This is the "setup_context" function.
+    2. You must tell us how to compute gradients during the backward pass.
+    This is the "backward" function.
+
+    ``setup_context_fn(ctx, inputs, output)`` runs during the forward pass.
+    Please save quantities needed for backward onto the ``ctx`` object via
```

Comment:
Not sure if I asked this in another PR but should we explicitly document the ctx here as being a custom Function ctx and link to that doc: https://pytorch.org/docs/stable/autograd.html#context-method-mixins ?